opal/aks-values.yaml

```diff
@@ -225,6 +225,10 @@ nginx:
   enabled: true
   appValues:
     namespace: *opalNS
+    condaChannelImage:
```

Comment:
Should these changes be included in this PR? It seems unrelated to the README

Reply:
I am not making a PR for a four-line fix. Especially not when the mandatory in-office policy means my PRs can only be reviewed on M/F. By condensing changes, I can get more updates through in a timely manner.

Reply:
Copy - yeah it does seem the process/policy interferes with work being done quickly, or for small changes to ever happen. Whatever gets the code released works for me; just doing my due diligence and this stood out to me